src/main/java/org/folio/dao/lines/PoLinesPostgresDAO.java

```diff
@@ -20,41 +20,40 @@
 
 
 public class PoLinesPostgresDAO implements PoLinesDAO {
-
-  private static final Logger logger = LogManager.getLogger(PoLinesPostgresDAO.class);
+  private static final Logger log = LogManager.getLogger();
 
   @Override
   public Future<List<PoLine>> getPoLines(Criterion criterion, DBClient client) {
+    log.trace("getPoLines, criterion={}", criterion);
     Promise<List<PoLine>> promise = Promise.promise();
-    client.getPgClient().get(PO_LINE_TABLE, PoLine.class, criterion, false, reply -> {
-        if (reply.failed()) {
-          logger.error("Retrieve POLs failed : {}", criterion.toString());
-          handleFailure(promise, reply);
-        } else {
-          List<PoLine> budgets = reply.result().getResults();
-          promise.complete(budgets);
-        }
-      });
+    client.getPgClient().get(PO_LINE_TABLE, PoLine.class, criterion, false, ar -> {
+      if (ar.failed()) {
+        log.error("getPoLines failed, criterion={}", criterion.toString());
```

Comment:
I see in some classes we are logging exception stack traces before handleFailures() and in this class not, maybe it makes sense to log here as well?

Reply:
Yes.